node-daemon/docs/RPC.md

```diff
@@ -356,41 +327,76 @@ returns the amount of coins owned by that delegation in that pool.
 Parameters:
 ```
 {
-    "pool_id": hex string,
-    "delegation_id": hex string,
+    "pool_address": string,
+    "delegation_address": string,
 }
 ```
 
 Returns:
 ```
-{
-    "val": number,
-} OR null
+EITHER OF
+     1) { "val": number }
```

Comment:
DecimalAmount is a good thing to return in this function. Returning Amount kind of doesn't make sense...

Reply:
Too many instances... this needs its own PR.